dist/fhir-client-angularjs.js

```diff
@@ -26326,6 +26326,11 @@ function isFakeOAuthToken(){
   return false;
 }
 
+function getUserLanguage() {
+  return ((navigator.languages && navigator.languages[0]) ||
```

Comment:
We usually check for the length of the array ```navigator.languages.lenght > 0```, is there a reason we are checking for only the first DOMString in the array?

Reply:
This check is saying that I want to get the most preferred language at index 0.
We can change the check to use `.length >0` before accessing the first index.

FYI 
https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/language
https://developer.mozilla.org/en-US/docs/Web/API/NavigatorLanguage/languages

These 2 APIs have different support for different browsers' versions.

Reply:
I'll remove this function since it's not the job of this library to do this.  We'll pass the header from our code to this library.